servers/rendering_server.cpp

```diff
@@ -2255,6 +2255,11 @@ void RenderingServer::_bind_methods() {
 	BIND_ENUM_CONSTANT(VIEWPORT_CLEAR_NEVER);
 	BIND_ENUM_CONSTANT(VIEWPORT_CLEAR_ONLY_NEXT_FRAME);
 
+	BIND_ENUM_CONSTANT(VIEWPORT_ENVIRONMENT_ENABLED);
```

Comment:
Make sure to bind them in the same order as the enum, so the numbers stay ordered in the docs.